Week-3/assignment2.js

```diff
@@ -0,0 +1,39 @@
+//! Apply
+
+var numbers = [2, 3, 8, 5, 1];
+
+var max = Math.max.apply(null, numbers);
+
+console.log("Max number is ", max);
+
+var min = Math.min.apply(null, numbers);
+
+console.log("Min number is ", min);
+
+//! Call
+
+function greet() {
+  var dummy = [this.human, "stands at", this.height].join(" ");
+
+  console.log(dummy);
+}
+
+var humans = {
+  human: "Avesh",
+  height: "6 ft tall",
+};
+
+greet.call(humans);
+
+//! Bind
+
+var module = {
+  x: 42,
+  getX: function () {
```

Comment:
@relhanakshay I don't think we need bind here. Calling module.getX will give me the right value. Can you define another object and use getX method in terms of that. Can you check ?

Reply:
@relhanakshay since this is single PR, i am not approving it. Can you complete assignment 4 and fix this issue as well.